srcpkgs/uxn/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'uxn'
+pkgname=uxn
+version=1.0
+revision=1
+makedepends="SDL2-devel"
+short_desc="Assembler and emulator for the Uxn stack-machine"
+maintainer="Eloi Torrents <eloitor@duck.com>"
+license="MIT"
+homepage="https://100r.co/site/uxn.html"
+distfiles="https://git.sr.ht/~rabbits/uxn/archive/${version}.tar.gz"
+checksum=29059ee288474e48fae4b9e755bbc8e1b392a9c7ac90449d7a6599385c145460
+
+do_build() {
+	CC="$BUILD_CC" CFLAGS="$XBPS_CROSS_CFLAGS" ./build.sh
```

Comment:
You don't want to mess with CC/CFLAGS, they are already set correctly for cross compiling.
You want to add `--no-run`, so it does not attempt to run the just compiled binaries.

The other problem is, the `build.sh` adds `-g0` to `CFLAGS`, which we don't want. The binaries will get stripped by `void-packages`

Reply:
Thank you. Is it a good idea to just put the 3 lines needed for compilation in `do_build` and ignore the `build.sh` file?

Reply:
Imo just patching it is better.
Vendoring parts of the script will add an additional burden of keeping it up to date.